src/ui/organisms/Eip7702Table.tsx

```diff
@@ -0,0 +1,329 @@
+    const logoPath = `/images/wallets/${walletId}.${row.original.wallet.metadata.iconExtension}`;
+    const defaultLogo = '/images/wallets/default.svg';
+    const walletUrl = `/${walletId}`;
+
+    return (
+      <div className='flex items-center'>
+        {/* Wallet Logo */}
+        <div className='flex-shrink-0 mr-3'>
+          <img
+            src={logoPath}
+            alt=''
+            className='w-6 h-6 object-contain'
+            onError={e => {
+              // Fallback for missing logos
+              e.currentTarget.src = defaultLogo;
+            }}
+          />
+        </div>
+        {/* Wallet Name */}
+        <a
+          href={walletUrl}
+          className='text-base font-medium hover:text-blue-600 hover:underline cursor-pointer'
+        >
+          {row.original.wallet.metadata.displayName}
+        </a>
+      </div>
+    );
+  };
+
+  return createCell;
+}
+
+interface WalletTypeCell {
+  typeString: string;
+  hasSmartWallet: boolean;
+}
+
+type CellValue = string | WalletTypeCell;
+
+export default function Eip7702Table(): React.JSX.Element {
+  const columnHelper = createColumnHelper<TableRow>();
+  const rankColumn = columnHelper.display({
+    id: 'rank',
+    header: '#',
+    cell: ({ row }): React.ReactNode => row.index + 1,
+  });
+  const walletNameColumn = columnHelper.accessor(
+    (row: TableRow): CellValue => row.wallet.metadata.displayName,
+    {
+      id: 'wallet',
+      header: 'Wallet',
+      cell: createWalletNameCell(),
+    },
+  );
+  const walletTypeColumn = columnHelper.display({
+    id: 'type',
+    header: 'Type',
+    cell: (info: CellContext<TableRow, unknown>): React.ReactNode => {
+      switch (info.row.original.typeFor7702) {
+        case WalletTypeFor7702.EIP7702:
+          return (
+            <div className='mt-1 flex flex-wrap gap-1'>
+              <EipStandardTag standard={eip7702} usePrefix={true} />
+            </div>
+          );
+        case WalletTypeFor7702.NON_7702_SMART_WALLET:
+          return (
+            <div className='mt-1 flex flex-wrap gap-1'>
+              <EipStandardTag standard={erc4337} usePrefix={true} />
+            </div>
+          );
+        case WalletTypeFor7702.NON_7702_EOA:
+          return <span>Non-7702 EOA</span>;
+        case WalletTypeFor7702.OTHER:
+          return <span>Non-7702</span>;
+      }
+    },
+  });
+  const walletContractColumn = columnHelper.display({
+    id: 'contract',
+    header: 'Contract',
+    cell: (info: CellContext<TableRow, unknown>): React.ReactNode => {
+      if (info.row.original.contract === 'NONE') {
+        return <span style={{ opacity: 0.5 }}>N/A</span>;
+      }
+
+      const contract = info.row.original.contract;
+
+      if (contract === 'UNKNOWN') {
+        return <span style={{ opacity: 1 }}>Unknown - Send PR!</span>;
+      }
+
+      return (
+        <ExternalLink
+          url={`https://etherscan.io/address/${contract.address}`}
```

Comment:
also a nit similar to the previous one, but maybe it would be nice to have a more global "EXPLORER_BASE_URL" const somewhere in a constant file, so we can reuse it.

Could be useful as such links might grow in the codebase + to support "multi chain" explorers.